site/content/docs/commands/task-run.en.md

```diff
@@ -56,6 +56,7 @@ Generally, the steps involved in task run are:
     --tag string                     Optional. The container image tag in addition to "latest".
 -n, --task-group-name string         Optional. The group name of the task. Tasks with the same group name share the same set of resources.
     --task-role string               Optional. The role for the task to use.
+     --acknowledge-secrets-access    Optional. Skips the secrets access permission grant question and directly grants the permission to secrets.
```

Comment:
We should copy our actual help menu here 😛 It says "Optional. Skip the confirmation question and grant access to the secrets specified by --secrets flag. 
This flag is useful only when '--secret' flag is specified"